### PR TITLE
Codacy fixes

### PR DESCRIPTION
**Pull request by pranav-sahasrabudhe**
Signed-off-by: Pranav Sahasrabudhe <Pranav.p.Sahasrabudhe@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

-----
[View rendered docs/Architectural-Overview.md](https://github.com/pranav-sahasrabudhe/cortx-test/blob/pranav-codacy-fixes/docs/Architectural-Overview.md)

### CODE REVIEW

File: docs/Architectural-Overview.md [hunk 1]
```diff
@@ -1,55 +1,87 @@
-
 ## Why moving from CTP and Avocado based test framework
-CTP OS and Avocado based test framework was used to develop automated test cases in Cortx R1/MVP. It had a binding to CentOS image which has Avocado and helper RPMs pre-installed for automation. Tests were written and run assuming prerequisites Python packages were installed. There was a hard dependency on CTP OS clients to run test automation. This framework was primarily aimed at testing firmware products and not for private cloud based solutions. It lacked parallel and distributed test execution capabilities at that point in time.
+
+CTP OS and Avocado based test framework was used to develop automated test cases in Cortx R1/MVP. It had a binding to
```
Comment (seagate-sarang-sawant) [inline docs/Architectural-Overview.md:3]: The html preview or page adjusts the line length and does the wrap.  Is there a need to make this  into multiple lines?

File: docs/Architectural-Overview.md [hunk 2]
```diff
@@ -121,175 +153,172 @@ Its a REST service which provides APIs to stores build wise test results and agg
...
+with automatic test configuration management, parallel test execution, reporting and storage efficiency.
 
-This framework can be deployed and used in two ways. In multi autobot setup mode and single autobot setup mode. 
+# Framework Components
 
- 
+The following diagram shows all components/modules present in framework.
 
-## Multi Autobot's setup: 
+* Master autobot: This module reads test execution data from test management tool. From this test execution data, it
```
Comment (seagate-sarang-sawant) [inline docs/Architectural-Overview.md:221]: Can you do branding change along with this work?  Introduction of Gliese and corbots.

**Review by seagate-sarang-sawant — CHANGES_REQUESTED** [review]

File: docs/Architectural-Overview.md [hunk 3]
```diff
@@ -121,175 +153,172 @@ Its a REST service which provides APIs to stores build wise test results and agg
...
+machines/ containers. Target Locking section in Framework components explains about how target is utilized for efficient
+test execution.
 
- 
+Test reporting data is saved by Restful Reporting Server to Mongo DB through Reporting client API. Reporting Client is
+called at the end of pytest test run with pytest reporting hook. This data is retrieved from Reporting server for
+further analytics and presentation.
 
- 
+# Single Auto bot's setup
```
Comment (seagate-sarang-sawant) [inline docs/Architectural-Overview.md:294]: Rebranding will fix line 294.

File: docs/Architectural-Overview.md (hunk 1 above)
Comment (pranav-sahasrabudhe) [reply docs/Architectural-Overview.md:3]: Looks like auto-formatting screwed up few things and fixed many things :( fixing by hand now.

**Comment by pranav-sahasrabudhe** [comment]
Created a new PR with latest code from main. Closing this older PR.

**Pull request closed without merging**